jormungandr/src/leadership/process.rs

```diff
@@ -262,7 +252,7 @@ impl Module {
                 // wait for the next epoch
 
                 tracing::debug!("no item scheduled, waiting for next epoch");
-                self.next_epoch_instant()
+                self.epoch_instant(Epoch(self.schedule.as_ref().unwrap().epoch().0 + 1))
```

Comment:
Is this always in the future?

Reply:
Not necessarily. For example, in the case that raised the bug this would result in an `Instant` that is before `Instant::now`